### PR TITLE
update gunk config with with JSON fix

### PR DESCRIPTION
**Pull request by favadi**
* generate go: fix inconsistent JSON tag names
* generate swagger: use JSON names in example
* docgen: use correct parameter fields in request/response examples
* docker is no longer required to run generate.sh

### CODE REVIEW

**Comment by karelbilek** [comment]
Notes about the generated markdown:

* this still includes the bug in the generated markdown, where `date_of_application` in the generated table is shortened to `application`. That's waiting for https://github.com/gunk/gunk/pull/288 
* note that, because of https://github.com/gunk/gunk/pull/283 , the generated CURL is now

  ```
  curl -X GET \
  	https:///v1/banks/{BankID}/accounts/public \
  	-H 'Authorization: Bearer USE_YOUR_TOKEN'
  ```

  because the .gunk files do have scheme, but do not have hostname.

* similarly, the line
  ```
  Host ``
  ```
  changes to
  ```
  Host `https://`
  ```

   Again because of the missing host, but not missing scheme

File: v1/account/all.swagger.json [hunk 1]
```diff
@@ -390,7 +390,7 @@
         },
         "parameters": [
           {
-            "name": "AccountID",
+            "name": "accountID",
```
Comment (karelbilek) [inline v1/account/all.swagger.json:401]: why is AccountID sometimes transformed to `accountID` and sometimes to `account_id`?

Comment (karelbilek) [reply v1/account/all.swagger.json:401]: oh, if it's a separate parameter, it's `accountID`, but if it's a field in the "JSON object", it's `account_id`?

Comment (favadi) [reply v1/account/all.swagger.json:401]: yes it is.

**Comment by karelbilek** [comment]
if it doesn't annoy you, please use latest gunk from master (after https://github.com/gunk/gunk/pull/288 ) so that we got markdown fixed

**Comment by karel-3d** [comment]
`go mod tidy` complains. Otherwise all looks good

**Comment by favadi** [comment]
updated gunk, merging.

**Pull request merged**